src/aux-server/aux-web/shared/scene/Game.ts

```diff
@@ -988,6 +1028,7 @@ export abstract class Game {
         this.renderer.clear();
         if (renderBackground) {
             this.mainSceneBackgroundUpdate();
+            this.mainScenePortalHDRAddressUpdate();
```

Comment:
@TroyceGowdy Please make the portalHDRAddress update even if the background was not rendered.